JciHitachi/connection.py

```diff
@@ -127,12 +108,10 @@ def login(self):
             "Accept": "application/json",
         }
 
-        login_req = httpx.post(
-            "{}{}".format(API_ENDPOINT, "UserLogin.php"),
+        login_req = self._client.post(
+            f"{API_ENDPOINT}UserLogin.php",
             json=login_json_data,
             headers=login_headers,
-            verify=API_SSL_CONTEXT if self._proxy is None else False,
-            proxy=self._proxy,
```

Comment:
I think this change is unnecessary as we can still feed `API_SSL_CONTEXT` to `httpx.post`. Let me know if I missed something.